R/zchunk_LA119.solar.R

```diff
@@ -29,10 +29,37 @@ module_gcam.usa_LA119.Solar_DISABLED <- function(command, ...) {
     # Load required inputs
     states_subregions <- get_data(all_data, "gcam-usa/states_subregions")
     NREL_us_re_capacity_factors <- get_data(all_data, "gcam-usa/NREL_us_re_capacity_factors")
-    NREL_us_re_technical_potential <- get_data(all_data, "gcam-usa/NREL_us_re_technical_potential")
+    NREL_us_re_technical_potential <- get_data(all_data, "gcam-usa/NREL_us_re_technical_potential")   #This data is not being used in this chunk
+
+    # Perform computations: Create scalars to scale capacity factors read in in the assumptions file in the energy folder to adjust for varying irradiance by state.
+
+    NREL_us_re_capacity_factors %>% gather(fuel,average,-State) %>%
+      filter (State=="Average") %>%
+      select (-State) -> Average_fuel
+
+    NREL_us_re_capacity_factors %>%
+      gather(fuel,value,-State) %>%
+      filter (State != "Average") %>%
+      full_join(Average_fuel, by = "fuel") %>%
+      mutate(scaler = value/average, sector = "electricity generation") %>%
+      select(State,sector,fuel, scaler) -> NREL_us_re_capacity_factors
+
+    states_subregions %>%
+      mutate(State = state_name) %>%
+      select(state,State) %>%
+      full_join(NREL_us_re_capacity_factors, by = "State") %>%
```

Comment:
Please put a comment that "state" is abbreviated state name and "State" is the full state name, and the final table uses abbreviated state name and "state" as the variable. I would have preferred using something like "state_abb" instead but ok for now.